schemas/ingest-metrics.v1.schema.json

```diff
@@ -40,7 +40,79 @@
             "type": "string"
           }
         },
-        "value": {},
+        "value": {
+          "anyOf": [
+            {
+              "title": "encoded_series_metric_value",
+              "type": "object",
+              "properties": {
+                "format": {
+                  "enum": ["array"]
```

Comment:
This can be const (I believe that is supported in the schema)

Reply:
Done